cogboard-app/src/main/kotlin/com/cognifide/cogboard/storage/docker/Validation.kt

```diff
@@ -0,0 +1,59 @@
+package com.cognifide.cogboard.storage.docker
+
+import com.cognifide.cogboard.CogboardConstants
+import io.vertx.core.json.JsonObject
+import java.lang.ClassCastException
+
+class Validation {
+    companion object {
+        fun validate(config: JsonObject): Boolean {
+            val boards = config.getJsonObject("boards")?.getJsonObject("boardsById")
+            return boards != null && validateBoards(boards)
+                    && config.getJsonObject("widgets")?.getJsonObject("widgetsById") != null
```

Comment:
consider naming the condition, divide it to sub-functions, e.g.:
`config.getJsonObject("widgets")?.getJsonObject("widgetsById") != null`
->
`hasWidgets(config)`